public/stylesheets/account.css

```diff
@@ -136,13 +135,20 @@ main {
   main {
     padding: 70px 8% 70px 5%;
   }
+  .change-password-div {
+    flex-direction: column;
+  }
+  .change-password-input {
+    margin-left: 0;
```

Comment:
Why not add this margin to label instead so there's no need to override it when the layout switches to column?